src/me.rs

```diff
@@ -942,9 +980,8 @@ fn full_search<T: Pixel>(
     let search_area = search_range_y.flat_map(|y| { search_range_x.clone().map(move |x| (y, x)) });
 
     let (cost, mv) = search_area.map(|(y, x)| {
-      let plane_org = p_org.slice(po);
-      let plane_ref = p_ref.slice(PlaneOffset { x, y });
-
+      let plane_org = p_org.region(Area::StartingAt { x: po.x, y: po.y });
+      let plane_ref = p_ref.region(Area::StartingAt { x, y });
```

Comment:
I think .region(Area::StartingAt { x, y } ) does not get panic when x or y is invalid value. For ex, if x is out of plane border (and crossing padded area, which is 22 pixels for 16x16 blk size at 1/4 ME). 
I tested this in estimate_motion_ss4( )
```
   full_search(
      x_lo - 200,
```
then encoder runs w/o any panic.

Reply:
Oh, good catch!

Initially, `PlaneRegion` used unsigned coordinates, so it only asserted max values, but then I needed to convert them to signed, but I forgot to assert min values.

I just fixed it.